Plots/MapProjections/NCL_native_2.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_proj_2_lg.py
+================
+
+This script illustrates the following concepts:
+   - Drawing filled contours over a mercator map
+   - Overlaying contours on a map without having lat,lon coordinates
+   - Drawing a map using the medium resolution map outlines
+   - Turning on map tickmark labels with degree symbols
+   - Selecting a different color map
+   - Turning off the addition of a longitude cyclic point
```

Comment:
I don't think this comment applies since by default a longitude cyclic point is not added
```suggestion
```